src/include/sof/audio/component.h

```diff
@@ -233,12 +233,14 @@ enum {
 #endif /* #if defined(__ZEPHYR__) && defined(CONFIG_ZEPHYR_LOG) */
 
 #define comp_perf_info(pcd, comp_p)					\
-	comp_info(comp_p, "perf comp_copy peak plat %u cpu %u",		\
+	comp_info(comp_p, "perf_cycle comp_copy peak plat %u cpu %u",		\
```

Comment:
We don't know all users of these performance traces, so I'd keep the names unchanged unless the semantics change.

Reply:
ok